openfe_gromacs/protocols/gromacs_md/md_methods.py

```diff
@@ -265,13 +269,16 @@ def get_gro_em_filename(self) -> list[pathlib.Path]:
         gro : list[pathlib.Path]
```

Comment:
Document that these can be Optional[list[pathlib.Path]], i.e. they can return None under certain conditions.

Reply:
Done